pkg/monitor/ovsdb.go

```diff
@@ -131,8 +134,22 @@ func (monitor *OVSDBMonitor) Run(stopChan <-chan struct{}) {
 	if err != nil {
 		klog.Fatalf("unable start ovsdb monitor: %s", err)
 	}
-
-	<-stopChan
+	for {
+		select {
+		case <-monitor.OvsdbReconnChan:
+			klog.Infof("reconnect ovsdb monitor")
+			monitor.ovsClient, err = ovsdb.ConnectUnix(ovsdb.DEFAULT_SOCK)
+			if err != nil {
+				klog.Fatalf("unable reconnect ovsdb socket: %s", err)
+			}
+			err := monitor.startOvsdbMonitor()
```

Comment:
Should we reset the monitor.ovsdbCache first?

Reply:
We will still reuse the cache in monitor instance，monitor.startOvsdbMonitor only reset monitor.ovsClient.